<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dannie</class>
 <widget class="QDialog" name="dannie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> Начальные настройки</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Введите координаты вершины:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>x0=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>y0=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>z0=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Введите коэффициенты сжатия:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>b=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>c=</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>a=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Построить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Введите цвет:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ввести координаты</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ввести координаты</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Чёрный</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Тёмно-зелёный</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Красный</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Синий</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Фиолетовый</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Жёлтый</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Зелёный</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_8">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Тёмно-красный</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Тёмно-синий</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_10">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Тёмно-фиолетовый</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
